### ISSUE TITLE
Adjust comment state in preparation for loading API data

### ISSUE THREAD

**Issue by kumar303** [mozilla/addons-code-manager#1082]
**For QA**: please check for regressions in the add comment feature (https://github.com/mozilla/addons-code-manager/issues/112)

In order to load comment data (needed for https://github.com/mozilla/addons-code-manager/issues/713), the comments state needs some structural adjustments.

**Comment by ioanarusiczki** [comment]
@kumar303 
Here are my test results on dev with FF69(Win10) on the Browse page:
- The "+" button to add a comment is present on every code line. 
- Many code lines are long - so the situation below is pretty common - user must scroll to read a longer comment (depends on the lenght of the code line). 

![what I can do](https://user-images.githubusercontent.com/33448286/66119883-4684f900-e5e2-11e9-9063-2799eb3582da.gif)

- I can't edit, delete or close the text box yet. (that's coming in the future https://github.com/orgs/mozilla/projects/66. 
- The messages are not available after a page refresh (known issue).

- I tried with Postman too - using the API 
https://addons-dev.allizom.org/api/v4/reviewers/addon/494431/versions/1532144/draft_comments/
My request JSON object was:

{   "comment": "sending this to line",
	"filename":"manifest.json",
	"lineno":1
}

I receive a 201 Created status with id:199. 

The frontend - https://code.addons-dev.allizom.org/en-US/browse/491860/versions/1528357/?path=manifest.json won't display it. I blame https://github.com/mozilla/addons-code-manager/issues/713 for this.

**Comment by kumar303** [comment]
Yeah, the wide form problem is tracked in https://github.com/mozilla/addons-code-manager/issues/997 . You could try commenting on a css file which typically has short lines. I haven't really tried to break the comment form so any testing there would be appreciated (emojis, empty comments, etc). Thanks!

**Comment by ioanarusiczki** [comment]
@kumar303 
- copy pasting emojis in the field works - the comment is saved and emojis are displayed. 
- if I save an empty comment the `Saving` state is shown and after it nothing happens - it should not allow empty comments, probably the `Save` should be disabled without at least 1 char typed in the filed.
- a long unbroken string will break the layout of the page.

![issues](https://user-images.githubusercontent.com/33448286/66294272-3fffc580-e8f1-11e9-9324-b6ccf8db9534.png)

I could file the 2 issues if they haven't been noticed yet.

**Comment by ioanarusiczki** [comment]
@kumar303 
I filed the issues here: 
https://github.com/mozilla/addons-code-manager/issues/1106
https://github.com/mozilla/addons-code-manager/issues/1108
This issue is verified on dev - FF69(Win10).

**Comment by kumar303** [comment]
Thanks for catching those!
